Jenkinsfile.github

```diff
@@ -193,5 +193,15 @@ pipeline {
                 }
              }
         }
+        stage("Test RPM Publish") {
```

Comment:
Why not comment out the entire stage?

Reply:
Risk aversion :) We still don't have a good answer for solid integration testing of our stuff, so I can see something like this coming back. The remaining steps here are no-op clean-ups.